plugins/modules/get_certificate.py

```diff
@@ -35,6 +35,10 @@
         - The port to connect to
       type: int
       required: true
+    server_name:
+      description:
+       - Server name used for SNI when hostname is an ip or is different from server name
```

Comment:
```suggestion
       - Server name used for SNI (L(Server Name Indication,https://en.wikipedia.org/wiki/Server_Name_Indication)) when hostname is an IP or is different from server name.
```

Reply:
(We should probably add an explanation/link for SNI in the module description as well.)